tools/@o3r/build-helpers/scripts/pr-labels.mjs

```diff
@@ -0,0 +1,27 @@
+import { EOL } from 'node:os';
+
+function getLabels(commitMessage) {
+  const commitLabels = [];
+
+  const lines = commitMessage.split(EOL);
+
+  lines.forEach((line) => {
+    if (line.match(/^\bfeat(ures?)?\b/)) {
```

Comment:
nitpick: `^` matches the start of the string so `^\b` should be equivalent to `^`